style.css

```diff
@@ -590,3 +627,794 @@
     justify-content: center;
   }
 }
+
+/* Desktop-desing */
+
+@media only screen and (min-width: 768px) {
+  .nav {
+    display: none;
+  }
```

Comment:
Kindly update your navbar styling to match the figma design better🙏

Your design:
<img width="507" alt="image" src="https://user-images.githubusercontent.com/37116322/236556011-7525b771-8d8c-4b1f-a343-c3759c6697c0.png">

Figma design:
<img width="460" alt="image" src="https://user-images.githubusercontent.com/37116322/236556108-86f0448e-114b-4df6-bb75-91160efbabe1.png">

A few things to note:
- The image icon is currently not displaying. Please check the file path associated with it
- Please make sure that the logo and navigation menu have the same colors, margins, padding and spacing as the figma design